package.json

```
@@ -2,9 +2,14 @@
   "name": "spectrum-css-monorepo",
   "version": "0.0.0",
   "private": true,
+  "type": "module",
   "scripts": {
-    "build": "gulp -LL",
-    "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
+    "build": "run-p build:11ty build:css",
+    "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true npx @11ty/eleventy --config=.eleventy.cjs --serve",
+    "build:11ty": "npx @11ty/eleventy --config=.eleventy.cjs",
```

11ty is installed as a dependency so `npx` should not be necessary. I'd also like to see the 11ty config inside the site workspace to keep all the assets together.

```suggestion
    "build:11ty": "@11ty/eleventy --config=./site/.eleventy.cjs",
```